bittensor/extrinsics/prometheus.py

```diff
@@ -96,7 +95,7 @@ def prometheus_extrinsic(
         bittensor.__console__.print(
             f":white_heavy_check_mark: [green]Prometheus already Served[/green]\n"
             f"[green not bold]- Status: [/green not bold] |"
-            f"[green not bold] ip: [/green not bold][white not bold]{net.int_to_ip(neuron.prometheus_info.ip)}[/white not bold] |"
+            f"[green not bold] ip: [/green not bold][white not bold]{net.ip_to_int(neuron.prometheus_info.ip)}[/white not bold] |"
```

Comment:
Why change this from `int_to_ip` to `ip_to_int`? 

The console logs will be much easier to read if we dont have to convert it back. Otherwise we will just see a long number that doesnt mean anything unless you run it through a conversion.

Reply:
This is the result of a long refactoring) Thank you! I went for coffee